src/extensions/update_manifest_handlers/steps_handler/src/steps_handler.cpp

```diff
@@ -127,6 +122,9 @@ ADUC_Result PrepareStepsWorkflowDataObject(ADUC_WorkflowHandle handle)
             }
             else
             {
+                ADUC_FileEntity entity;
```

Comment:
would it better to have at outermost scope so that ADUC_FileEntity_Uninit could be called at done: label like in develop branch?